infra/mock_api/template/rds.tf

```diff
@@ -1,6 +1,40 @@
-# rds config goes here
-# need db user and pw
-# need security group
-# use postgres instance
+data "aws_ssm_parameter" "db_username" {
+  name = "POSTGRES_USER"
+}
+data "aws_ssm_parameter" "db_pw" {
+  name = "POSTGRES_PASSWORD"
+}
+data "aws_ssm_parameter" "db" {
+  name = "POSTGRES_DB"
+}
+data "aws_ssm_parameter" "db_host" {
+  name = "DB_HOST"
+}
+resource "aws_security_group" "rds" {
+  description = "allows connections to RDS"
+  name        = "rds-instance"
+  vpc_id      = module.constants.vpc_id
 
-# RDS IAM user
\ No newline at end of file
+  ingress {
+    from_port       = 5432
+    to_port         = 5432
+    protocol        = "tcp"
+    security_groups = ["${aws_security_group.allow-api-traffic.id}", "${aws_security_group.handle-csv.id}"]
+  }
+
+}
+
+resource "aws_db_instance" "mock_api_db" {
```

Comment:
you'll usually want to enable deletion protection for your DB instance, not sure if you want to do that in this case for a demo project.

Reply:
I thought it would be a bit much for a demo project, but if its considered a best practice, I'll enable it